### DISCUSSION TITLE
Resource Locations

### DISCUSSION THREAD

**Discussion by linxule** [Ledgerback/DGSF#8] [Q&A]
Can we maintain a page with links to various resources outside the repo? 

For example, could you @kylefox1 provide the link to the Airtable base mentioned in https://github.com/Ledgerback/DGSF/discussions/3#discussioncomment-8804422 ? I am struggling with finding the table/sheet and bases.

**Comment by kylefox1** [comment]
I will add a table/base on NocoDB for it, but this might take some time as I migrate the bases.

**Reply by kylefox1** [reply]
You can find the resources catalogue in the Catalogue-Resources_Tools-2023_10_21 table.

You can find it here: https://data.ledgerback.xyz/dashboard/#/nc/view/4b2a1582-8a00-4983-83ae-b8c5d4a4107a

**Reply by linxule** [reply]
@vincenzospaghetti could we integrate this into our resource wiki/page?

**Comment by vincenzospaghetti** [comment]
i'll make a list :)

**Comment by linxule** [comment]
For Netnography, here are some relevant literature:

1. Marchetti, A., & Puranam, P. (2022). Organizational cultural strength as the negative cross-entropy of mindshare: a measure based on descriptive text. *Humanities and Social Sciences Communications*, *9*(1), 1-14.
2. Vicinanza, P., Goldberg, A., & Srivastava, S. B. (2023). A deep-learning model of prescient ideas demonstrates that they emerge from the periphery. *PNAS nexus*, *2*(1), 275.
3. Shrestha, Y. R., & He, V. F. (2022). Integrating multi-modal data and machine learning for entrepreneurship research. Available at: https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4204889
4. Park, S., Piezunka, H., & Dahlander, L. (2023). Coevolutionary lock-in in external search. *Academy of Management Journal.*
5. Miric, M., Jia, N., & Huang, K. G. (2023). Using supervised machine learning for large‐scale classification in management research: The case for identifying artificial intelligence patents. *Strategic Management Journal*, 44(2), 491-519.
6. Corritore, M., Goldberg, A., & Srivastava, S. B. (2020). Duality in diversity: How intrapersonal and interpersonal cultural heterogeneity relate to firm performance. *Administrative Science Quarterly*, 65(2), 359-394.
